src/components/mdMenu/mdMenu.vue

```diff
@@ -63,81 +63,73 @@
         this.menuContent.classList.remove('md-size-' + size);
       },
       removeLastDirectionMenuContentClass(direction) {
-        this.menuContent.classList.remove('md-direction-' + direction.replace(' ', '-'));
+        this.menuContent.classList.remove('md-direction-' + direction.replace(/ /g, '-'));
       },
       addNewSizeMenuContentClass(size) {
         this.menuContent.classList.add('md-size-' + size);
       },
       addNewDirectionMenuContentClass(direction) {
-        this.menuContent.classList.add('md-direction-' + direction.replace(' ', '-'));
+        this.menuContent.classList.add('md-direction-' + direction.replace(/ /g, '-'));
       },
-      getBottomRightPos() {
+      getPosition(vertical, horizontal, align) {
```

Comment:
I merged all 4 functions into 1 to avoid dupplication. I think it looks clean now.